pkg/util/maven/maven.go

```diff
@@ -178,17 +178,24 @@ func Run(ctx Context) error {
 
 	Log.Debug("About to start parsing the Maven output")
 
+	structuredLogs := HasStructuredLog()
+
 	for scanner.Scan() {
```

Comment:
Would trying parsing the first line and falling back to unstructured be a solution? It seems it'd be a more robust heuristic than relying on `MAVEN_OPTS` and `logback.configurationFile` that does not guarantee structured logging is actually enabled.

Reply:
Thanks @astefanutti ! I think you are right. 

One thing that I did not consider - even in the original implementation is: what if we have an output to the maven log that outputs non-structured bits? In this case, it should go to the normal logger instead of the normalization part. 

And, in fact, there are a few scenarios where this could happen (ie.: for example, if the JDK prints a warning, a misbehaved maven plugin bypass the Maven logger, etc). 

Let me re-adjust the implementation based on your feedback before we merge this.